nemoguardrails/library/injection_detection/flows.v1.co

```diff
@@ -0,0 +1,23 @@
+define bot refuse to respond
+  "I'm sorry, I can't help you with that."
+
+
+define subflow reject injection
+  """
+  Reject injection attempts
+  """
+  $injection = execute reject_injection(source="output", text=$bot_message)
```

Comment:
@Pouyanpi source is not used in mitigate_injection, It should throw invalid argument error

source is a way of controlling a common interface used for both input and output rails. I think it is not needed here because we only use these actions for output rails.